### PR TITLE
Bach BQ: Remove `COALESCE` from `JsonBigQueryAccessorImpl.get_value`

### PR DESCRIPTION
**Pull request by KathiaBarahona**
The coalesce is generating big query expressions as sometimes we might have expressions for nested jsons.

### CODE REVIEW

File: bach/bach/series/series_json.py [hunk 1]
```diff
@@ -505,13 +505,17 @@ def get_value(self, key: str, as_str: bool = False) -> Union['SeriesString', 'TS
...
+        # if accessed value is a json, this will return NULL since JSON_VALUE
+        # is used only for scalars
         value_expression = Expression.construct(
-            '''JSON_VALUE({}, '$."{}"')''',
-            self._series_object,
-            Expression.raw(key)
+            f'REGEXP_REPLACE({{}}, {{}}, {{}})',
+            json_expression,
+            Expression.raw('r\'(^"|"$)\''),
+            Expression.string_value(''),
```
Comment (KathiaBarahona) [inline bach/bach/series/series_json.py:514]: @thijs-obj Decided to just replace the start/end quotes just for having the same behavior as we have with Postgres. If we just return NULL when the value to be extracted is a json, then we need to add a condition on the tests... so I'm not sure if there's another way to go. Also tried `FORMAT('%t', ....)` and `TRIM(...,  '"')` but didn't worked for some edge cases (when having nested quotes).

File: bach/bach/series/series_json.py [hunk 2]
```diff
@@ -505,13 +505,17 @@ def get_value(self, key: str, as_str: bool = False) -> Union['SeriesString', 'TS
         if not as_str:
             return self._series_object.copy_override(expression=json_expression)
         from bach.series import SeriesString
+        # if accessed value is a json, this will return NULL since JSON_VALUE
```
Comment (thijs-obj) [inline bach/bach/series/series_json.py:508]: Comment doesn't make sense, as we don't use JSON_VALUE anymore

File: bach/bach/series/series_json.py [hunk 3]
```diff
@@ -505,13 +505,17 @@ def get_value(self, key: str, as_str: bool = False) -> Union['SeriesString', 'TS
...
+        # is used only for scalars
```
Comment (thijs-obj) [inline bach/bach/series/series_json.py:509]:
```suggestion
        # JSON_QUERY will return a string containing valid json. i.e. if the queried value is a string, then
        # it will be quoted. Here we remove the quotes
```

**Review by thijs-obj — APPROVED** [review]
Left a suggestion for a comment, please have a look to see what you think.

Code is okay tho :+1:

File: bach/bach/series/series_json.py (hunk 2 above)
Comment (KathiaBarahona) [reply bach/bach/series/series_json.py:508]: yeh, was doing the opposite. Forgot to remove that
